packages/client/hmi-client/src/components/documents/tera-document.vue

```diff
@@ -10,6 +10,7 @@
 		@close-preview="emit('close-preview')"
 		:hide-header="documentView === DocumentView.PDF"
 		:stretch-content="documentView === DocumentView.PDF"
+		:show-sticky-header="documentView === DocumentView.PDF"
```

Comment:
You could have removed the `:hide-header` prop that was causing your issue.

Reply:
@YohannParis There's 2 notions of a header on the tera-asset component,
This is what the hide-header "hides" 
<img width="1402" alt="Screenshot 2023-07-06 at 8 58 23 AM" src="https://github.com/DARPA-ASKEM/Terarium/assets/33158416/b72a18c5-0371-4dda-9b92-11f9aff46278">

The below is the sticky header which shows when you scroll down and has slightly different layout
<img width="1471" alt="Screenshot 2023-07-06 at 8 58 59 AM" src="https://github.com/DARPA-ASKEM/Terarium/assets/33158416/ed10187b-fd2f-46ff-a3e8-51834105d979">

For viewing the PDF page we just want the sticky header to be visible at all times rather than the entire header section

Reply:
damn, then could you rename `hide-header` to `hide-intro`? to be clearer

Reply:
@YohannParis good idea, this has been addressed!